### ISSUE TITLE
Focus styles in dark backgrounds may be insufficient

### ISSUE THREAD

**Issue by jasmussen** [WordPress/wporg-theme-directory#135]
Focus a link in the footer:

![Image](https://github.com/WordPress/wporg-theme-directory/assets/1204802/490395d0-161f-436b-9e7d-af97616f11e0)

The focus outline may have insufficient contrast. Can it be changed to `currentColor`, either globally or just contextually?
